pyproject.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 packages = [{include = "finch", from = "src"}]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
```

Comment:
We use `obj[*idxs]` which was introduced in `3.11` so I think we need to drop `3.10`.

Reply:
I see, yes I changed this in CI but not in poetry